{"name": "Принятие заказа с различными невалидными ID курьеров", "status": "passed", "steps": [{"name": "Попытаться принять заказ с невалидным ID курьера: ", "status": "passed", "steps": [{"name": "Принять заказ 546573 курьером ''", "status": "passed", "parameters": [{"name": "order_id", "value": "546573"}, {"name": "courier_id", "value": "''"}], "start": 1758527540203, "stop": 1758527540602}], "start": 1758527540203, "stop": 1758527540602}, {"name": "Проверить что запрос завершился ошибкой", "status": "passed", "start": 1758527540602, "stop": 1758527540602}], "parameters": [{"name": "invalid_courier_id", "value": "''"}], "start": 1758527540202, "stop": 1758527540604, "uuid": "96c0acd5-7c26-4de1-b2d2-289075bde5c7", "historyId": "4409f057984de48866aed05d3c5372e9", "testCaseId": "4409f057984de48866aed05d3c5372e9", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_various_invalid_courier_ids", "labels": [{"name": "epic", "value": "Order Management"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Accept Order API"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "9396-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}